runtime/vm/jvmfree.c

```diff
@@ -439,3 +447,29 @@ cleanUpClassLoader(J9VMThread *vmThread, J9ClassLoader* classLoader)
 }
 #endif /* J9VM_GC_DYNAMIC_CLASS_UNLOADING */
 
+#if defined(J9VM_OPT_CRIU_SUPPORT) && defined(LINUX)
+I_32
+disclaimAllClassMemory(J9VMThread *currentThread)
```

Comment:
also this should take J9JavaVM instead of J9VMThread

Reply:
How can I find the current thread for the tracepoint by iterating the J9JavaVM mainThread?

Reply:
You dont need to do that. Just add `noenv` to the tracepoint, and it will automatically find it.